src/App.js

```diff
@@ -1,26 +1,90 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react'
 import './App.css';
+import contacts from './contacts.json'
+
+const App = () => {
+  //Iteration 1 | Display 5 Contacts
+  const fiveContacts = contacts.slice(0, 5);
+  const [contactsState, setcontactsState] = useState(fiveContacts);
+
+  //Iteration 2 | Add New Random Contacts
+  const addRandomContacts = () => {
+    const randomContacts = Math.floor(Math.random() * contacts.length);
```

Comment:
A random contact is added. However, this code results in duplicate contacts. :) 
Perhaps, we can check whether a contact is added before, or splice the contact from the contacts array when added.